src/main.cpp

```diff
@@ -7,21 +7,63 @@
 
 #include "ecu_lua_script.h"
 #include "electronic_control_unit.h"
+#include "j1939_simulator.h"
 #include "ecu_timer.h"
 #include "utilities.h"
 #include <string>
+#include <thread>
+#include <signal.h>
```

Comment:
Include `<csignal>` and `<cstdio>` here instead of signal.h and stdio.h, since we're talking c++

Reply:
Fixed